git-helper.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
We should prefer `#!/usr/bin/env sh` for more environment-agnostic scripts, though this is not a big deal, since we're assuming developers are using Macs for the most part.